requirements.txt~

```diff
@@ -0,0 +1,3 @@
+soldier==0.1
```

Comment:
It is ignoring the file.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	deleted:    requirements.txt~

no changes added to commit (use "git add" and/or "git commit -a")
dwaipayan@dwaipayan-HP-ENVY-15-Notebook-PC:~/Bot/osdc-bot$ git add .
warning: You ran 'git add' with neither '-A (--all)' or '--ignore-removal',
whose behaviour will change in Git 2.0 with respect to paths you removed.
Paths like 'requirements.txt~' that are
removed from your working tree are ignored with this version of Git.

* 'git add --ignore-removal <pathspec>', which is the current default,
  ignores paths you removed from your working tree.

* 'git add --all <pathspec>' will let you also record the removals.

Run 'git status' to check the paths you removed from your working tree.